boards/arm/96b_argonkey/96b_argonkey.dts

```diff
@@ -46,6 +46,28 @@
 	};
 };
 
+&clk_hse {
+	clock-frequency = <DT_FREQ_M(16)>;
+	status = "okay";
+};
+
+&pll {
+	div-m = <8>;
+	mul-n = <84>;
+	div-p = <2>;
+	div-q = <8>;
+	clocks = <&clk_hse>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>;
+	clock-frequency = <DT_FREQ_M(84)>;
+	ahb-prescaler = <1>;
+	apb1-prescaler = <2>;
+	apb2-prescaler = <2>;
```

Comment:
Could be interesting keeping the comment here ("APB2 clock is fixed at 42MHz to prevent known SPI/I2S bug")

Reply:
Ok. Done